workshops/weather_app/README.md

```diff
@@ -0,0 +1,455 @@
+---
+name: 'Weather App'
+description: 'Making a Weather App using HTML, CSS and JS(no framework involved)'
+author: '@gautamjajoo'
+---
+
+# Weather App
+
+Are you a beginner in Javascript and don't know how to start working with APIs? 
+
+If yes, you are at the right place. Being a beginner at Js and beginning to work with APIs could be very painful.
+But with a simple project we would learn some basic elements of `JS` and I am sure till the end of the workshop you will get a hang of 
+how APIs work.  
+
+In this App we would take an input of a city from the user and display the weather details(Current temp, Min/Max Temp etc) of that particular city.
+
+Before we start building our app let's see how the final version would look like:
+
+![main](https://cloud-ktt2vamjn.vercel.app/0screencapture-file-home-gautamjajoo-desktop-card-index-html-2020-10-09-00_34_48.png)
+
+Isn't it amazing? Get your cup of coffee and 20 minutes of your time to build an awesome weather app. 
+
+The [Source Code](https://github.com/gautamjajoo/BasicWeather)  &  [Demo](https://WeatherApp.gautamjajoo.repl.co) are hyperlinked here.
+
+## 0. What could you expect till the end of the workshop?
+
+Till the end of the workshop you would have a good understanding of how to deal with APIs. You could also expand this knowledge to be
+creative and implement new APIs in your projects.
+
+Also, you would learn some nice `CSS` tricks for making beautiful card designs and basic elements/properties of `JS`.
+
+## 1. The Prerequisites
+Some basic understanding of `HTML,CSS and Javascipt/JSON`.
+
+Don't be upset if you don't know any of the above, we will make sure you learn something new till the end of the workshop.
+
+## 2. Setting up the project on Repl.it
+
+Repl.it is an powerful online IDE(integrated development environment) where you can edit, build and collabrate to make some beautiful projects.

```

Comment:
```suggestion
Repl.it is a powerful online IDE(integrated development environment) where you can edit, build, and collaborate to make some beautiful projects.
```suggestion
Repl.it is a powerful online IDE(integrated development environment) where you can edit, build, and collaborate to make some beautiful projects.

```
```